src/components/BookList.js

```diff
@@ -1,17 +1,23 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import Book from './Book';
 import Form from './Form';
+import { fetchBooks } from '../redux/books/booksSlice';
 
 function BookList() {
+  const dispatch = useDispatch();
+
   const books = useSelector((state) => state.books.books);
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, [dispatch]);
```

Comment:
- [ ] Good job but there is a bug in your project that you need to fix before proceeding. When I add new books in your project, and then if I try to remove any of them, they all get disappeared and then I have to refresh the page to fetch them. it happens specifically whenever there are no books and then you add any number and then the first time you want to remove. After refreshing and getting all the books back, the removing functionality works fine.
here is the video

https://github.com/naji-97/sec-book-store/assets/96683843/1c26b618-0ab7-4b53-9ec5-d550ee45f43b